### ISSUE TITLE
Unsupported sensor

### ISSUE THREAD

**Issue by duczz** [pnbruckner/ha-illuminance#72]
Hey, whats my problem here? normally the default value that is be setted should be use?
Sometimes it works and sometimes not. maybe when the weather integration reloads?

```
Dieser Fehler wurde von einer benutzerdefinierten Integration verursacht

Logger: custom_components.illuminance.sensor
Quelle: custom_components/illuminance/sensor.py:421
Integration: Illuminance (Dokumentation, Probleme)
Erstmals aufgetreten: 18:02:29 (1 Vorkommnisse)
Zuletzt protokolliert: 18:02:29

Illuminance Home: Unsupported sensor weather.penthouse: not a number, no attribution attribute, or doesn't exist
```
![image](https://github.com/user-attachments/assets/7e5e9f55-af6e-4723-a08f-73a3294eb475)

**Comment by pnbruckner** [comment]
What version are you using? If you haven't tried 5.5.0, please do so and let me know if you still have the problem.

**Comment by duczz** [comment]
I'm using 5.5.0

**Comment by pnbruckner** [comment]
Does `weather.penthouse` exist? If so, can you share its state, including attributes?

**Comment by pnbruckner** [comment]
Thanks! What weather integration does this entity come from? Normally, I would expect to see an `attribution` attribute, but this one doesn't appear to have one.

Originally, the `attribution` attribute was used to determine how to map the weather condition (i.e., state) to a division factor, but since there is now a standard list of conditions, it's not needed as much anymore. Really, it only matters for the AccuWeather and ecobee weather entities. I think I can remove the requirement for this attribute and just use the standard mapping when it isn't present.

**Comment by duczz** [comment]
Its comes from https://github.com/jaydeethree/Home-Assistant-weatherdotcom

**Comment by pnbruckner** [comment]
Thanks.

It's unfortunate that it does not add the `attribution` attribute to the `weather` entity. It does add it to the `sensor` entities, but it's unfortunate it does not have a "condition" sensor. It seems this integration differs from other weather integrations I've seen in these two respects.

Still, I'm working on loosening the restrictions surrounding the `attribution` attribute. I should have a beta for you to try soon.

**Comment by duczz** [comment]
Perfect thanks you 👍🏻
Should be ask the other Integration creator that he add this missing "Standard" setting?

**Comment by pnbruckner** [comment]
@duczz, please try version 5.6.0b0 and let me know if it solves your problem.

**Comment by pnbruckner** [comment]
Yep, that's what I expect. It's not an error, it's just a warning, and it should function normally.

**Comment by duczz** [comment]
Perfect, so thank you for this fast update 😁👍🏻

**Comment by duczz** [comment]
@pnbruckner a short question to this problem, make it sence to remove this log entry to have a clean log? (i click always on it when i check my logs ^^)

**Comment by pnbruckner** [comment]
Yeah, I suppose I could make it a debug statement. That way it would not normally appear, but if you were wondering, you could enable debug to see it.

**Comment by pnbruckner** [comment]
Give version 5.6.0b1 a try.

**Comment by duczz** [comment]
Perfect, works great. thanks you 👍🏻👍🏻

**Comment by pnbruckner** [comment]
Fix is fully released in 5.6.0.
